doc/users/next_whats_new/imshow_extent_units.rst

```diff
@@ -8,19 +8,25 @@ can now be expressed with units.
 
     import matplotlib.pyplot as plt
     import numpy as np
-    from matplotlib.dates import HourLocator, DateFormatter
+    from matplotlib.dates import HourLocator, ConciseDateFormatter
     from matplotlib.ticker import AutoMinorLocator
 
-    fig, ax = plt.subplots()
+    fig, ax = plt.subplots(layout='constrained')
     date_first = np.datetime64('2020-01-01', 'D')
     date_last = np.datetime64('2020-01-11', 'D')
 
     arr = [[i+j for i in range(10)] for j in range(10)]
 
     ax.imshow(arr, origin='lower', extent=[1, 11, date_first, date_last])
 
-    ax.yaxis.set_major_formatter(DateFormatter('%d/%m/%y:- %H00hours'))
-    ax.yaxis.set_major_locator(HourLocator(byhour=[0, 6, 12, 18, 24]))
+    locator = HourLocator(byhour=[0, 12, 24])
```

Comment:
If you get rid of 24 here does it still render the same? My thinking is that 0 and 24 ticks overlap between one day and the next day.

Reply:
It still renders the same. I kept it because it was in the original example, but it is a little confusing. I will remove it.